src/StringContainsExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SQLite_net
+{
+	/// <summary>
+	/// Class for Contains extension with StringComparison option
+	/// </summary>
+	public static class StringContainsExtension
```

Comment:
Please rename to `SQLiteStringExtensions` and move it into the file SQLite.cs. Please delete this file.